src/arrayMethodSort.js

```diff
@@ -5,7 +5,38 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    if (compareFunction === undefined) {
+      for (let index = 0; index < this.length; index++) {
+        for (let nextIndex = index; nextIndex < this.length; nextIndex++) {
+          const element = String(this[index]);
+          const nextElement = String(this[nextIndex]);
+
+          if (element > nextElement) {
+            const temp = this[index];
+
+            this[index] = this[nextIndex];
+            this[nextIndex] = temp;
+          }
+        }
+      }
+
+      return this;
+    }
```

Comment:
```suggestion
 
```
By implementing a default compare function you can make your code smaller and avoid duplication of logic